doc/sfp-cmis/CMIS-custom-SI-settings.md

```diff
@@ -0,0 +1,281 @@
+                                                        "OutputEqPostCursorTargetRx2": 8,
+                                                        "OutputEqPostCursorTargetRx3": 8,
+                                                        "OutputEqPostCursorTargetRx4": 8,
+                                                        "OutputEqPostCursorTargetRx5": 8,
+                                                        "OutputEqPostCursorTargetRx6": 8,
+                                                        "OutputEqPostCursorTargetRx7": 8,
+                                                        "OutputEqPostCursorTargetRx8": 8
+                                                }
+                                        },
+                                        "Default": {
+                                                "OutputEqPreCursorTargetRx": {
+                                                        "OutputEqPreCursorTargetRx1": 9,
+                                                        "OutputEqPreCursorTargetRx2": 9,
+                                                        "OutputEqPreCursorTargetRx3": 9,
+                                                        "OutputEqPreCursorTargetRx4": 9,
+                                                        "OutputEqPreCursorTargetRx5": 9,
+                                                        "OutputEqPreCursorTargetRx6": 9,
+                                                        "OutputEqPreCursorTargetRx7": 9,
+                                                        "OutputEqPreCursorTargetRx8": 9
+                                                }
+
+                                        }
+                                }
+                        },
+                        "25,28,30": {
+                                "100G_SPEED": {
+                                        "Default": {
+                                                "OutputAmplitudeTargetRx": {
+                                                        "OutputAmplitudeTargetRx1": 7,
+                                                        "OutputAmplitudeTargetRx2": 7,
+                                                        "OutputAmplitudeTargetRx3": 7,
+                                                        "OutputAmplitudeTargetRx4": 7,
+                                                        "OutputAmplitudeTargetRx5": 7,
+                                                        "OutputAmplitudeTargetRx6": 7,
+                                                        "OutputAmplitudeTargetRx7": 7,
+                                                        "OutputAmplitudeTargetRx8": 7
+                                                }
+                                        }
+                                }
+                        }
+                },
+                "PORT_MEDIA_SETTINGS": {
+                        "31": {
+                                "100G_SPEED": {
```

Comment:
Can you please clarify how are you going to get this lane speed? Where is it stored?

Reply:
The lane speed in JSON will be compared with lane_speed generated in CMIS FSM using module speed divided by host lane count. This lane speed will be passed to the api which we set module SI settings